src/components/BottomBar/ReplyBoard.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+
+class Reply extends React.Component{
+
+    constructor(props, context){
+        super(props, context);
+        this.state={
+            isEditing: false,
+            reply: {
+                id: this.props.reply.id,
```

Comment:
Thay vì dùng this.props c dùng props thôi cũng được rồi. hàm constructor nó được truyền vào tham số props rồi mà :D